Automattic/wp-calypso
# Jetpack Backup: add some additional Tracks events on the backup browser

Related to https://github.com/Automattic/jetpack-backup-team/issues/489

## Proposed Changes

* Add `calypso_jetpack_backup_browser_view_file` event when a user clicks on a file in the backup browser, including the [file type](https://github.com/Automattic/wp-calypso/blob/6bef95373d9f4986ce44a04354a2042c57b00f17/client/my-sites/backup/backup-contents-page/file-browser/util.ts#L11).
* Add `calypso_jetpack_backup_browser_preview_file` event when user clicks to a file and the preview renders, including the [file type](https://github.com/Automattic/wp-calypso/blob/6bef95373d9f4986ce44a04354a2042c57b00f17/client/my-sites/backup/backup-contents-page/file-browser/util.ts#L11).
* Add `calypso_jetpack_backup_browser_preview_file_sensitive_click` event when user clicks to `Show preview` when trying to preview a sensitive file (wp-config.php).

## Why are these changes being made?


These specific Tracks events will help us validate the feature usage of the backup browser feature.

## Testing Instructions



* Spin up a Jetpack Cloud live branch
* Open the Network tab in your developer tools
* Navigate to Jetpack Cloud > Backup
* On any available backup, click on `Actions (+)`, then on `View files` to access the backup browser.
* Click on any directory, table, image, or code files.
* Ensure you see a request to `t.gif` with the `calypso_jetpack_backup_browser_view_file` once you click on any file (not directories) including the file type. It should also apply if you click on plugins and themes.
* Ensure you see a request to `t.gif` with the `calypso_jetpack_backup_browser_preview_file` once the file preview renders.
* Now try clicking on `wp-config.php`. You should see the warning and the `Show preview` button. Click it!
* Ensure you see a request to `t.gif` with the `calypso_jetpack_backup_browser_preview_file_sensitive_click` once you click `Show preview`.

## Pre-merge Checklist
